### DISCUSSION TITLE
Mod settings not appearing on online leaderboards

### DISCUSSION THREAD

**Discussion by apollo-dw** [ppy/osu#19145] [Q&A] [answered]
Local score: 
![image](https://user-images.githubusercontent.com/83023433/179354917-b044a696-b43a-4289-b695-b6f1cd931c57.png)

Online score:
![image](https://user-images.githubusercontent.com/83023433/179354929-66706554-f67f-43a6-8cda-433d86d8e921.png)

**Comment by Theighlin** [comment, marked as answer]
An issue for this has been opened minutes before this one at #19144.
